Makefile

```diff
@@ -75,6 +76,7 @@ SHADER_OBJS = \
 	$(SRCDIR)/shaders/fog_vec4_wy.inl \
 	$(SRCDIR)/shaders/fog_vec4_wzyx.inl \
 	$(SRCDIR)/shaders/fog_vec4_xyzw.inl \
+	$(SRCDIR)/shaders/mul_by_one_vertex_shader.inl \
```

Comment:
Sorry, I missed that the shader file was renamed but the makefile wasn't updated.

Reply:
missed the makefile - have updated the commit